study.md

```diff
@@ -27,17 +27,17 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+Not a source or a rendered page.
 ```
```

Comment:
by not a source I assume you mean that it's not the source code or what we see when we 'View Source'. If so you're right!